modules/terraform-zscc-bastion-azure/variables.tf

```diff
@@ -49,7 +49,10 @@ variable "instance_size" {
 }
 
 locals {
-  unsupported_regions = ["chinanorth", "chinaeast", "china east", "china north"]
+  unsupported_regions          = ["chinanorth", "chinaeast", "china east", "china north", "usgovirginia"]
```

Comment:
Do we use this local variable any more?

Reply:
yeah, i left this around mainly for just for china as it has some nuance issues with os_disk types in my testing

storage_account_type = contains(local.unsupported_regions, lower(var.location)) ? "Standard_LRS" : "Premium_LRS"